packages/core/src/api/resolvers/shop/shop-order.resolver.ts

```diff
@@ -353,6 +352,28 @@ export class ShopOrderResolver {
         );
     }
 
+    @Transaction()
+    @Mutation()
+    @Allow(Permission.UpdateOrder, Permission.Owner)
+    async addItemsToOrder(
+        @Ctx() ctx: RequestContext,
+        @Args() args: MutationAddItemsToOrderArgs & ActiveOrderArgs,
```

Comment:
`@Relations` doesn't work here, because the query has a nested `addItemsToOrder{ order: {}}` field. I suppose it's only a performance optimization? it now uses the default relations.

Reply:
yes, the Relations decorator is not that smart yet. This should be OK. The entire feature is a huge perf optimization anyway.